pkg/controller/core/localqueue_controller.go

```diff
@@ -128,7 +128,7 @@ func (r *LocalQueueReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 	if meta.IsStatusConditionTrue(cq.Status.Conditions, kueue.ClusterQueueActive) {
-		err = r.UpdateStatusIfChanged(ctx, &queueObj, metav1.ConditionTrue, "Ready", "Can submit new workloads to clusterQueue")
+		err = r.UpdateStatusIfChanged(ctx, &queueObj, metav1.ConditionTrue, kueue.ClusterQueueActive, "Can submit new workloads to localQueue")
```

Comment:
should this be ClusterQueueActive, or LocalQueueActive?

Reply:
And are we okay with this changing from "Ready" to "Active"?

Reply:
Done